### DISCUSSION TITLE
Can I use another git server for extension hosting?

### DISCUSSION THREAD

**Discussion by iusgit** [quarto-dev/quarto-cli#2862] [Q&A]
Hi,
I have a self-hosted gitea server and develop there an quarto extension...

Using the URL of that didn't work. Is it in general possible to use a git server different than Github?

**Comment by cscheid** [comment]
You can always ask people to download a tarball and `quarto install local-path-to-extension.tar.gz`

**Reply by iusgit** [reply]
Tried that:
Downloaded https://github.com/quarto-ext/latex-environment/archive/refs/tags/v0.1.0.zip , then created a book project and did
`quarto install extension ~/Downloads/latex-environment-0.1.0.zip`

Result:
```
[✓] Unzipping
    Found 0 extensions.
ERROR: Invalid extension
The extension staged at /var/folders/gl/7hy3xp816p50g_1nppx2gjl80000gp/T/quarto-sessiond75e3396/75e86c3e/a7104967/staged does not provide any valid extensions.
```
The same happened also for other extensions.
```
quarto -V
1.2.209
```

**Reply by dragonstyle** [reply]
I'll take a closer look at why that is failing locally - it should work! In the meantime, I was able to install this extension using:

`quarto add https://github.com/quarto-ext/latex-environment/archive/refs/tags/v0.1.0.zip`

**Reply by dragonstyle** [reply]
Our next pre-release build of v1.2 should fix this issue and support local install of a downloaded GitHub zip archive - thank you for raising this!

**Reply by iusgit** [reply]
Is that v1.2.219 or thereafter?

**Reply by dragonstyle** [reply]
It is actually just there now - v1.2.221.
